vim/plugin-managers.md

```diff
@@ -0,0 +1,26 @@
+# Plugin Managers
+
+Today I learned that vim has two main plugin managers:
+
+1. Tim Pope<sup>1</sup>'s [Pathogen][1]
+1. gmarik's [Vundle][2]
```

Comment:
Very cool - learn something new erryday. I wonder if it's a best practice though? 

[The list section](http://daringfireball.net/projects/markdown/syntax#list) of the Daring Fireball guide seems to suggest that it is a "lazy" practice. However, I also definitely see how allows one to manage large and high churn lists easily without the fear of breaking list ordering.

Some food for thought!